### PR TITLE
putting faq updates live

### PR DESCRIPTION
**Pull request by jospoortvliet**

### CODE REVIEW

**Comment by jospoortvliet** [comment]
@phil-davis I'll put this live already but your input is still very welcome - I'd like to make sure I have no errors in here ;-)

@RealRancor your input also welcome, what do you think?

File: page-faq.php [hunk 1]
```diff
@@ -157,11 +159,22 @@
...
 </ul>
 
+<a name="calendarcontacts"></a>
+<h3>Why are Calendar and Contacts (or another app) not shipped with the zip file, are they not part of ownCloud?</h3>
+<p>We have four types of apps: Core, Official, Approved and Experimental.Calendar and Contacts are a bit of an issue as, right now, they are not super well maintained and we probably should downgrade them to Approved. If they were core parts of ownCloud users would expect that they get the same attention as ownCloud core. But these apps are developed mostly by students in their free time and thus are not as stable, don’t scale to large instances and don’t get such swift attention to bugreports as the core of ownCloud which is maintained by people who are paid full-time by customers. If a company (or more than one) steps up to pay ownCloud, Inc. for maintenance of the calendar/contact, or if
```
Comment (RealRancor) [inline page-faq.php]: Missing point between: Experimental.Calendar

The sentence also just stops with ", or if " and then the different types of app are listed.

**Comment by RealRancor** [comment]
Havn't checked if for spelling but :+1: for the statement itself.

It might be also fitting in here:

http://owncloud.10557.n7.nabble.com/Information-for-users-of-Weuxels-Gentoo-owncloud-client-Repository-td16533.html#a16539

as the user there is moving away from ownCloud mainly because of the broken calendar and contacts stuff.

File: page-faq.php (hunk 1 above)
Comment (phil-davis) [inline page-faq.php]: It just needs a space after the end of "Experimental." - "Calendar" starts a new sentence.
Also I think "bugreports" is 2 words "bug reports.

File: page-faq.php [hunk 2]
```diff
@@ -157,11 +159,22 @@
...
 </ul>
 
+<a name="calendarcontacts"></a>
+<h3>Why are Calendar and Contacts (or another app) not shipped with the zip file, are they not part of ownCloud?</h3>
+<p>We have four types of apps: Core, Official, Approved and Experimental.Calendar and Contacts are a bit of an issue as, right now, they are not super well maintained and we probably should downgrade them to Approved. If they were core parts of ownCloud users would expect that they get the same attention as ownCloud core. But these apps are developed mostly by students in their free time and thus are not as stable, don’t scale to large instances and don’t get such swift attention to bugreports as the core of ownCloud which is maintained by people who are paid full-time by customers. If a company (or more than one) steps up to pay ownCloud, Inc. for maintenance of the calendar/contact, or if 
+<ul>
+	<li><strong>Core</strong> means part of ownCloud core, like sharing, gallery, external storage, encryption etc. Being part of core, problems would block releases and the employees of ownCloud, Inc. maintain most of the components.</li>
```
Comment (phil-davis) [inline page-faq.php]: "large number of users" -> "large numbers of users"

File: page-faq.php [hunk 3]
```diff
@@ -157,11 +159,22 @@
...
+	<li><strong>Official</strong> are apps which are quite well maintained and important for ownCloud. They include the Calendar and Contacts. Being 'official', we deeply care about them and will try to have them available for new releases on release day, but they are still developed by volunteers. This also means there is no commercial support available from ownCloud, Inc. This often also means that these apps do not work well on systems with large number of users or large amounts of data.</li>
```
Comment (phil-davis) [inline page-faq.php]: "large number of users" -> "large numbers of users"

**Comment by phil-davis** [comment]
I could only notice just those 2 little things I have commented on.

File: page-faq.php (hunk 1 above)
Comment (RealRancor) [inline page-faq.php]: Yes, i mean space not point. :-)

**Pull request merged**